docs/resources/sql_global_config.md

```diff
@@ -17,13 +19,32 @@ resource "databricks_sql_global_config" "this" {
 }
 ```
 
+### Azure example
+
+For Azure you should use the `data_access_config` to provide the service principal configuration. You can use the Databricks SQL Admin Console UI to help you generate the right configuration values.
+
+```hcl
+resource "databricks_sql_global_config" "this" {
+  security_policy      = "DATA_ACCESS_CONTROL"
+  data_access_config = {
+    "spark.sql.session.timeZone" : "UTC",
+    "spark.hadoop.fs.azure.account.auth.type.lakehouse.dfs.core.windows.net" : "OAuth",
+    "spark.hadoop.fs.azure.account.oauth.provider.type.lakehouse.dfs.core.windows.net" : "org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider",
+    "spark.hadoop.fs.azure.account.oauth2.client.id.lakehouse.dfs.core.windows.net" : "...",
+    "spark.hadoop.fs.azure.account.oauth2.client.secret.lakehouse.dfs.core.windows.net" : "{{secrets/your-secret-scope-name/your-secret-name}}",
```

Comment:
replace with `"{{secrets/${local.secret_scope}/${local.secret_key}}}"`, so it's more clear what's variable and what's not